R-package/tests/testthat/test_basic.R

```diff
@@ -41,7 +41,7 @@ test_that("train and predict softmax", {
     data = as.matrix(iris[, -5L])
     , label = lb
     , num_leaves = 4L
-    , learning_rate = 0.1
+    , learning_rate = 0.05
```

Comment:
how is this change related?

Reply:
or, sorry, let me rephrase...why was it necessary to reduce the learning rate for this test?

Reply:
@jameslamb I just have a try. In previous, the output is multiplied by `1/2`. while currently, it is `3/4` (with num_class=4). So I reduce the learning rate a little bit.

Reply:
or can you help me to fix this test?

Reply:
oooo I see, thanks! It looks like you were able to fix the test, sorry for my delayed reply